treeherder/etl/job_loader.py

```diff
@@ -239,6 +239,22 @@ def _get_log_references(self, job):
                 "url": logref["url"],
                 "parse_status": "parsed" if "steps" in logref else "pending"
             })
+        log_references.extend(self._get_errorsummary_log_references(job))
+        return log_references
+
+    def _get_errorsummary_log_references(self, job):
+        log_references = []
+        if "jobInfo" in job:
+            ji = job["jobInfo"]
+            if "links" in ji:
+                links = ji["links"]
+                for link in links:
+                    if ("linkText" in link and
+                        link["linkText"].endswith("_errorsummary.log")
+                        and "url" in link):
+                        log_references.append({"name": "errorsummary_json",
+                                               "url": link["url"],
+                                               "parse_status": "pending"})
```

Comment:
Sure, if you prefer. I will note that it's probably less performant since we end up creating a bunch of unused objects every time. But I guess we don't really care about performance.